### PR TITLE
updated: metrics-related docs with respect to new metric updates

### LINKED ISSUE #239: Update metric-related docs with the respect to recent metric updates in Kyverno
The metric design was recently upgraded in Kyverno to deal with cardinality explosion.
Refer: https://github.com/kyverno/kyverno/pull/2157

Update the metric-related docs to comply with the new metrics.

### PR DESCRIPTION
**Pull request by yashvardhan-kukreja**
Signed-off-by: Yashvardhan Kukreja <yash.kukreja.98@gmail.com>

closes #239 

Updates the metric-related docs to comply with the recent metric re-design in Kyverno.
Refer: https://github.com/kyverno/kyverno/pull/2157

### CODE REVIEW

**Comment by chipzoller** [comment]
And this is only going to make it into 1.4.2, correct?

File: content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+kyverno_admission_requests_total
+
+**Metric Value**
+
+Counter - An only-increasing integer representing the count of admission requests associated with a sample.
+
+## Use cases
+
+* The cluster admin wants to know how many admission requests were triggered in the last 24h how active Kyverno has been.
```
Comment (chipzoller) [inline content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md:17]: "...the last 24h and how active..." ?

File: content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Admission Requests
+description: This metric can be used to track the number of admission requests which were triggered as a part of Kyverno.
```
Comment (chipzoller) [inline content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md:3]: "These metrics..." ? The description here is a summary of the content in the entire page.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Minor comments.

File: content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md (hunk 2 above)
Comment (yashvardhan-kukreja) [reply content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md:3]: This is a single metric actually. Hence, "this metric"

File: content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md (hunk 1 above)
Comment (yashvardhan-kukreja) [reply content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md:17]: Aah, my bad. Will resolve this.

**Comment by yashvardhan-kukreja** [comment]
> And this is only going to make it into 1.4.2, correct?

Yep

File: content/en/docs/Monitoring Kyverno With Prometheus Metrics/policy-execution-latency.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
 ---
 title: Policy Rule Execution Latency
-description: This metric can be used to track the latencies associated with the execution/processing of the individual rules whenever they evaluate incoming resource requests or execute background scans.
+description: This metric can be used to track the latencies associated with the execution/processing of the individual rules whenever they evaluate incoming resource requests or execute background scans. This metric can be further aggregated to present latencies at policy-level.
```
Comment (chipzoller) [inline content/en/docs/Monitoring Kyverno With Prometheus Metrics/policy-execution-latency.md:3]: "...latency at the policy level."

File: content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Filter Labels
+
+| Label                        | Allowed Values                                         | Description                                                                       |
+| ---------------------------- | ------------------------------------------------------ | --------------------------------------------------------------------------------- |
+| resource\_name               |                                                        | Name of the resource which is being evaluated as a part of this admission review. |
+| resource\_kind               | "Pod", "Deployment", "StatefulSet", "ReplicaSet", etc. | Kind of this resource                                                             |
+| resource\_namespace          |                                                        | Namspace in which this resource lies                                              |
+| resource\_request\_operation | "create", "update", "delete"                           | If the requested resource is being created, updated or deleted.                   |
```
Comment (chipzoller) [inline content/en/docs/Monitoring Kyverno With Prometheus Metrics/admission-requests.md:27]: "...created, updated, or deleted." (Oxford comma)

**Review by chipzoller — CHANGES_REQUESTED** [review]
Couple minor things then done.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**